include/os/Makefile.am

```diff
@@ -1,3 +1,6 @@
 if BUILD_LINUX
 SUBDIRS = linux
 endif
+if BUILD_FREEBSD
+#SUBDIRS = freebsd
+endif
```

Comment:
It looks like this should be uncommented and `include/os/freebsd/Makefile.am` populated.